hq/app/schedule/Dynamo.scala

```diff
@@ -27,13 +27,35 @@ class Dynamo(client: AmazonDynamoDB, tableName: Option[String]) extends Attribut
       "error"
   }
 
+  def scan: Seq[Map[String, AttributeValue]] = {
+    client.scan(new ScanRequest().withTableName(table)).getItems.asScala.map(_.asScala.toMap)
+  }
+
+  def scanAlert(): Seq[IamAuditUser] = {
```

Comment:
I forgot to mention this at the time but I think it would be god to long term add some error handling here  so that if the thing in the database  is e.g. missing 'alerts' then we return a HTTP error explaining what's gone wrong. but initially you could wrap this in a Try and in the failure case just return an empty list - I think that might be slightly better than just having an exception